assets/src/edit-story/components/library/panes/media/mediaPane.js

```diff
@@ -86,28 +85,21 @@ const DEFAULT_WIDTH = 150;
 function MediaPane(props) {
   const {
     state: { media, isMediaLoading, isMediaLoaded, mediaType, searchTerm },
-    actions: {
-      loadMedia,
-      reloadMedia,
-      resetMedia,
-      setMediaType,
-      setSearchTerm,
-      uploadVideoFrame,
-    },
+    actions: { fetchMedia, setMediaType, setSearchTerm, uploadVideoFrame },
   } = useMedia();
+
   const {
     allowedMimeTypes: {
       image: allowedImageMimeTypes,
       video: allowedVideoMimeTypes,
     },
   } = useConfig();
+
   const {
     actions: { insertElement },
   } = useLibrary();
 
-  useEffect(loadMedia);
-
-  const onClose = resetMedia;
+  const onClose = fetchMedia;
```

Comment:
Before this reset the search and filter terms on dialog close. Does this happen now as well?